index.html

```diff
@@ -0,0 +1,217 @@
+        <h3 class="header-text">My Recent Works</h3>
+        <div class="separator"></div>
+    </div>
+
+    <section id="works-section">
+
+    </section>
+
+    <section class="about-me" id="about">
+        <div id="about-text">
+            <div class="bg-img1"></div>
+            <div class="header-style">
+                <h2 class="header-text">About me</h2>
+            </div>
+            <div class="p-style">
+                <p> I can help you build a product, feature or website Look
+                    through some of my work and experience! If you like what
+                    you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+            <div class="bg-img2"></div>
+            <a class="pro-btn resume-btn" href="#contact-section">Get My Resume</a>
+
+            <div class="skills-list">
+                <div class="lang tech-sk mt-2">
+                    <div class="lg">
+                        <img src="./images/lang.png" alt="languages mastered">
+                    </div>
+                    <h4>Language</h4>
+                    <ul>
+                        <li>JavaScript</li>
+                        <li>Ruby</li>
+                        <li>Html </li>
+                        <li>CSS</li>
+                    </ul>
+                </div>
+                <div class="framework tech-sk">
+                    <div class="frm">
+                        <img src="./images/frame.png" alt="frameworks used">
+                    </div>
+                    <h4>Framework</h4>
+                    <ul>
+                        <li>Bootstrap</li>
+                        <li>Ruby on rails</li>
+                        <li>RSpec</li>
+                        <li>Capybara</li>
+                        <li>Webpack</li>
+                    </ul>
+                </div>
+                <div class="skill-set tech-sk">
+                    <div class="sk">
+                        <img src="./images/skillz.png" alt="skills">
+                    </div>
+                    <h4>Skills</h4>
+                    <ul>
+                        <li>Codekit</li>
+                        <li>GitHub</li>
+                        <li>Codepen</li>
+                        <li>Gitlab</li>
+                        <li>Terminal</li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="contact-section" id="contact">
+        <h2 id="greeting">Hello</h2>
+        <p>I'm always interested in hearing about new projects,
+            so if you'd like to chat please get in touch.</p>
+        <form action="/send-https://formspree.io/f/mpzbbknz" method="POST" class="contact-form" id="contact-details">
+            <label>
+                <input type="text" id="name" name="name" placeholder="Full name" required maxlength="30">
+            </label>
```

Comment:
[Compliment] I didn't know one could avoid giving attributes to a ``label`` tag by wrapping it over the input 😄  🎉